pup-0005.md

```diff
@@ -0,0 +1,148 @@
+developers for a few reasons: (1) GPLv2 and LGPLv2.x are licenses that
+many Red Hat developers continue to prefer; (2) projects led by Red
+Hat developers accept contributions under the project license (as
+opposed to an asymmetrical contributor license agreement); and (3)
+traditionally Red Hat developers have exercised broad discretion on
+project licensing policy.
+
+Pulp is a good candidate for adoption of the CCRC at the project
+level. Pulp is a GPLv2-licensed community project of commercial
+significance to Red Hat, and Pulp has been making significant efforts
+to build a community around the project, which over time will
+encourage contributions to the project from various copyright holders.
+
+## Common Cure Rights Commitment, version 1.0
+
+Common Cure Rights Commitment, version 1.0
+ 
+Before filing or continuing to prosecute any legal proceeding or claim
+(other than a Defensive Action) arising from termination of a Covered
+License, we commit to extend to the person or entity ('you') accused
+of violating the Covered License the following provisions regarding
+cure and reinstatement, taken from GPL version 3. As used here, the
+term 'this License' refers to the specific Covered License being
+enforced.
+
+    However, if you cease all violation of this License, then your
+    license from a particular copyright holder is reinstated (a)
+    provisionally, unless and until the copyright holder explicitly
+    and finally terminates your license, and (b) permanently, if the
+    copyright holder fails to notify you of the violation by some
+    reasonable means prior to 60 days after the cessation.
+
+    Moreover, your license from a particular copyright holder is
+    reinstated permanently if the copyright holder notifies you of the
+    violation by some reasonable means, this is the first time you
+    have received notice of violation of this License (for any work)
+    from that copyright holder, and you cure the violation prior to 30
+    days after your receipt of the notice.
+
+We intend this Commitment to be irrevocable, and binding and
+enforceable against us and assignees of or successors to our
+copyrights.
+
+Definitions
+
+'Covered License' means the GNU General Public License, version 2
+(GPLv2), the GNU Lesser General Public License, version 2.1
+(LGPLv2.1), or the GNU Library General Public License, version 2
+(LGPLv2), all as published by the Free Software Foundation.
+
+'Defensive Action' means a legal proceeding or claim that We bring
+against you in response to a prior proceeding or claim initiated by
+you or your affiliate.
+
+'We' means each contributor to this repository as of the date of
+inclusion of this file, including subsidiaries of a corporate
+contributor.
+
+This work is available under a [Creative Commons Attribution-ShareAlike 4.0 International license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## Displaying the CCRC
+
+A file named COMMITMENT (or some alternative name) will be added to
```

Comment:
Can the "or some alternative name" be removed? To be adopted we need the name to be named here in the PUP.

Reply:
I pushed a commit with this revision to the PR so this is fixed. See that specific change here: https://github.com/richardfontana/pups/commit/99fcd35e1cc396a1ba5a34555f093304dd07a333